doc/workday/signature-suite.md

```diff
@@ -0,0 +1,81 @@
+
+## Contents
+  * [Abstract](#abstract)
+  * [Contents](#contents)
+  * [Specification](#specification)
+    - [ID](#id)
+    - [Type](#type)
+      + [Verification Key](#verification-key)
+    - [Canonicalization Algorithm](#canonicalization-algorithm)
+    - [Digest Algorithm](#digest-algorithm)
+    - [Signature Algorithm](#signature-algorithm)
+  * [Prior Art](#prior-art)
+  * [Implementation Notes](#implementation-notes)
+  * [References](#references)
+
+## Specification
+According to the [Linked Data Signatures](https://w3c-dvcg.github.io/ld-signatures/) document, a _Linked Data Signature_
+
+>  is designed to be easy to use by developers and therefore strives to minimize the amount of information one has to remember to generate a signature. 
+
+The document outlines five properties that are required to comprise a Signature Suite: **id**, **type**, **canonicalizationAlgorithm**, **digestAlgorithm**, and **signatureAlgorithm**.
+
+This specification is intended to work only on JSON data.
+
+### ID
+> A URL that identifies the signature suite. For example: https://w3id.org/security/v1#Ed25519Signature2018.
+
+The URL which hosts. It will be on a publicly facing GitHub repository, or hosted on the credentials site such as: [https://credentials.workday.com/docs/specification/v1.0/signature-suite.md](https://credentials.workday.com/docs/specification/v1.0/signature-suite.md).
+
+### Type
+> The value SignatureSuite.
+   
+WorkEd25519Signature2020. This type is used to represent proofs.
+
+## Verification Key
+
+WorkEd25519VerificationKey2020. This type is used to represent keys that specifically generate WorkEd25519Signature2020 signatures.
+
+### Canonicalization Algorithm
+> A URL that identifies the canonicalization algorithm to use on the document. For example: https://w3id.org/security#URDNA2015.
+
+This signature suite uses [Draft 17 of the JSON Canonicalization Scheme (JCS)](https://tools.ietf.org/id/draft-rundgren-json-canonicalization-scheme-17.html), currently holding the __Informational__ status, by the [IETF Network Working Group](https://datatracker.ietf.org/drafts/current/). Future versions of this specification may use later versions of JCS as the spec progresses.
+
+### Digest Algorithm
+> A URL that identifies the message digest algorithm to use on the canonicalized document. For example: https://www.ietf.org/assignments/jwa-parameters#SHA256
+
+SHA-512, as a part of the [Ed25519](https://tools.ietf.org/html/rfc8032) signing process, as outlined in [IETF RFC 6234](https://tools.ietf.org/html/rfc6234).
+
+### Signature Algorithm
+> A URL that identifies the signature algorithm to use on the data to be signed. For example: http://w3id.org/security#ed25519 
+
+Ed25519 as outlined in [IETF RFC 8032](https://tools.ietf.org/html/rfc8032).
```

Comment:
only Ed25519? Consider adding support for JOSE... https://github.com/w3c-ccg/lds-jws2020

Ed25519 is great, but there are other key types and signature formats... and you will need to create a new suite for each if you make this only Ed25519... You will need to register each of them again, and the PRs to get them added to DID Core / W3C might take months....

Reply:
Also you probably need to specify "Detached" or not detached "EdDSA"... I expect "Detached" given thats what every othter LD suite has used.